### DISCUSSION TITLE
Change color scheme on local database site so i can tell which tab is running locally

### DISCUSSION THREAD

**Discussion by AvidDabbler** [supabase/supabase#11127] [Feature Requests]
# Feature request

## Is your feature request related to a problem? Please describe.

I keep almost pushing changes to production. It would be nice if the local version had a different color scheme. Maybe a yellow or an orange instead of the green for the highlight color in the app and tab icon. That way I know yellow is local green is production.

## Describe the solution you'd like

Change the color scheme from local to production to make it easier to tell them apart. 

## Describe alternatives you've considered

NA

## Additional context

<img width="585" alt="image" src="https://user-images.githubusercontent.com/8471756/206193402-69e33dee-db0d-44cb-a5b9-71d3e51ac128.png">

**Comment by vpribish** [comment]
I keep tripping on this too.  how hard can it be to just put a different color in, like, the top nav bar on the CLI tool - or make it a setting or something, anything!

<img width="1142" height="116" alt="Screenshot 2026-02-18 at 12 08 39 PM" src="https://github.com/user-attachments/assets/a27f5931-39b0-454d-aef4-2acdef2930ee" />
